File: bubble_sorts/bubble_sort.py
def bubble_sort(arr):
    for i in range(0, len(arr)):
        for j in range(0, len(arr) -i -1):
            if arr[j] > arr[j + 1]:
                tmp = arr[j]
                arr[j] = arr[j + 1]
                arr[j + 1] = tmp

File: bubble_sorts/bubble_sort.rb
def bubble_sort(arr)
  (arr.length).times do |i|
    ((arr.length) -1 -i).times do |j|
      if arr[j] > arr[j + 1]
        tmp = arr[j]
        arr[j] = arr[j + 1]
        arr[j+1] = tmp
      end
    end
  end
end
